ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -315,7 +382,8 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
      * "reviewRequested" flag, which depend on the workspace state & edit mode.
      */
     createInitialWorkspaceState(): Workspace {
-      let workspace: Workspace = this.props.workspace;
+      // copy the props into a new object so our modifications here don't affect them
+      let workspace: Workspace = {...this.props.workspace};
```

Comment:
Fixes an existing bug: Duplication screen showed "Duplicate of Duplicate of My Workspace"